configs/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "strictNullChecks": true,
     "noImplicitOverride": true,
     "noEmitOnError": false,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
It is really annoying during development. Every time you want to test something, it won't build because you have an unused variable somewhere, which is logical because you are still building the feature.

Reply:
@ozcanxbreeze  Can this config be dynamic based on whether you are debugging or doing a 'normal' build? Cause in the build pipeline you would like it to fail with unused variables I think.

Reply:
You could make a different tsconfig.json for the build pipeline. But then you have to change the package.json for every package in this repo and set up a correct pipeline. That is outside the scope of this pbi. I turned this rule on again. 

Worth making a pbi to set this up